network-api/networkapi/minisites/templates/minisites/blocks/iframe_block.html

```diff
@@ -0,0 +1,9 @@
+{% load wagtailcore_tags %}
+
+<iframe
+    width="{{ block.value.width }}"
```

Comment:
Is this going to make it easy to break responsiveness on pages?

Reply:
iframes the bane of responsiveness to begin with: they don't "fit" their content like a div or the like does, they *must* have a specified width and height attribute. We could set those in percentages but that's about the max in terms of responsiveness I know of.

Reply:
Right, I think a default of 100% would be useful. @gvn, thoughts?

Reply:
Agree. 💯

Reply:
What minimal height should they be?